src/iface-modbus.c

```diff
@@ -125,15 +237,15 @@ int iface_modbus_read_data(int reg_addr, int bit_offset, knot_value_type *out)
 			tmp.val_byte |= byte_tmp[i] << i;
 		break;
 	case TYPE_U16:
-		rc = connection_interface.read_u16(modbus_ctx, reg_addr,
+		rc = read_u16(modbus_ctx, reg_addr,
 						   &tmp.val_u16);
```

Comment:
Can you check if this parameter can be fitted in the previous line?
If it can't please allign the first column with the function's first parameter column